src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1029,6 +1053,64 @@ private Task<ICosmosDBAccount> CreateCosmosDbAsync()
                     .CreateAsync(cts.Token));
 
+        private async Task<IMySQLManagementClient> CreateAndProvisionMySQLServerAsync()
+        {
+            var mySQLManagementClient = new MySQLManagementClient(azureCredentials)
+            { SubscriptionId = configuration.SubscriptionId };
+
+            // Create Server.
+            await Execute(
+                $"Creating MySQL On Azure Server Instance: {configuration.MySQLServerName}...",
+                () => mySQLManagementClient.Servers.CreateAsync(
+                     configuration.ResourceGroupName, configuration.MySQLServerName,
+                     new ServerForCreate(
+                         new ServerPropertiesForDefaultCreate(
+                             administratorLogin: "cromwell",
+                             administratorLoginPassword: "Pa$$word123"),
+                         configuration.RegionName)));
+
+            // Create a firewall rule for current machine's external IP.
+            var externalIpString = new WebClient().DownloadString("http://icanhazip.com").Replace("\\r\\n", "").Replace("\\n", "").Trim();
+            var externalIp = IPAddress.Parse(externalIpString);
+            await Execute(
+                $"Creating Firewall Rule...",
+                () => mySQLManagementClient.FirewallRules.CreateOrUpdateAsync(
+                    configuration.ResourceGroupName, configuration.MySQLServerName, "AllowLocalIP",
+            new FirewallRule(externalIp.ToString(), externalIp.ToString())));
+            
+            // Create Database.
+            await Execute(
+                $"Creating MySQL On Azure Server Databse: cromwell_db...",
+                () => mySQLManagementClient.Databases.CreateOrUpdateAsync(
+               configuration.ResourceGroupName, configuration.MySQLServerName, "cromwell_db",
+               new Database()));
+
+            // Execute queries
+            ConsoleEx.WriteLine("Executing scripts on cromwell_db.");
+            var connectionString = $"Database=cromwell_db;Data Source={configuration.MySQLServerName}.mysql.database.azure.com;User Id=cromwell@{configuration.MySQLServerName};Password=Pa$$word123";
+            using (var connection = new MySqlConnection(connectionString))
+            {
+                var initScript = new MySqlScript(connection, Utility.GetFileContent("scripts", "mysql", "init-user.sql"));
+                var unlockChangeScript = new MySqlScript(connection, Utility.GetFileContent("scripts", "mysql", "unlock-change-log.sql"))
+                {
+                    Delimiter = "$$"
+                };
+                await connection.OpenAsync();
+                await initScript.ExecuteAsync();
+                await unlockChangeScript.ExecuteAsync();
+            }
+
+            // Update the cromwell-application.conf file with server name.
+            // TODO: FIX??? Find and replace somehow not writing but working in code????
```

Comment:
help plz

Reply:
The method `WritePersonalizedFilesToStorageAccountAsync()` is writing the changes to this file to the VM. This code here sends a copy to the VM (if the VM exists), which is overwritten by that other method, or the copy created here doesn't go anywhere. This change needs to be made in that method I indicated.